src/PageLayout/Notification/NotificationDetailsController.js

```diff
@@ -0,0 +1,132 @@
+                    }
+                },
+                // ADDED FOR DUMMY DATA
+                donatingAgencyDetails: {
+                    name: 'Local Point',
+                    primaryContact: {
+                        name: 'Andrea Benson',
+                        email: 'AndreaIsCool@uw.edu',
+                        phone: '206-586-9876',
+                        position: 'Manager'
+                    },
+                    address: {
+                        street1: '1201 NE Campus Pkwy',
+                        street2: '',
+                        city: 'Seattle',
+                        state: 'WA',
+                        zipcode: 98105,
+                        officeNo: '220'
+                    }
+                },
+                delivererGroup: {
+                    // same as receivingAgency above:
+                    requested: 'R8BAHrxdkfQoAmfWvGa1OJmjQP43',  // uid-key of a DG
+                    // OR
+                    pending: [
+                        'R8BAHrxdkfQoAmfWvGa1OJmjQP43',  // uid-key of all DGs
+                    ],
+                    // OR
+                    claimed: 'R8BAHrxdkfQoAmfWvGa1OJmjQP43',  // uid-key of a DG (once a DG claims)
+                    // ADDED FOR DUMMY DATA
+                    deliverers: [
+                        // {
+                        //     name: 'Alice',
+                        //     email: 'alice@uw.edu',
+                        //     phone: '123-789-4560'
+                        // },
+                        // {
+                        //     name: 'Chris',
+                        //     email: 'chris@uw.edu',
+                        //     phone: '456-123-0789'
+                        // }
+                    ]
+                },
+                requestTimeStamp: 1518753363763,
+                spawnedDeliveries: [
+                    // individual deliveries that were created to fulfill this delivery request
+                    '-L5RkIS0CSPuXpkewaqA'
+                ]
+            }
+        };
+    }
+
+    componentDidMount(){
+        //TODO: Query for notification based on {this.props.notifcation.content}
+    }
+
+    render() {
+        return (
+            <div className="popup-wrapper recurring">
+                <img className="close" src={close} alt="close" onClick={this.props.closePopUp} />
+                {
+                    this.props.notification.type === NotificationType.RECURRING_PICKUP_REQUEST && 
+                    <RecurringRequestController
+                        details={this.state.details}
+                        closePopUp={this.props.closePopUp.bind(this)}
+                        account={this.props.account}/>
+                }
+                {
+                    this.props.notification.type === NotificationType.RECURRING_PICKUP_CONFIRMED &&
+                    <h1>Recurring pickup: {this.props.notification.content} is confirmed.</h1>
+                }
```

Comment:
add another case for the 3 failure types (see PR overall comment):
```
if (this.props.notification.type === NotificationType.RECURRING_PICKUP_REJECTED_RA ||
    this.props.notification.type === NotificationType.RECURRING_PICKUP_REJECTED_DG ||
    this.props.notification.type === NotificationType.RECURRING_PICKUP_UNAVAILABLE) {
    return <RecurringRequestCanceled ... />
}
```
(if you use `switch/case`, can use "fall through" for these 3 cases)